index.rst

```diff
@@ -13,13 +13,93 @@
 Abstract
 ========
 
-This document describes the concept of operations for running the Auxiliary Telescope in Survey Mode, which will become the primary operational mode for AuxTel. The Survey Mode utilizes semi-autonomous, scheduler-driven operations and the roles, responsibilities, and expectations for this mode are defined. 
+This document describes the concept of operations for running the Auxiliary Telescope in Survey Mode, which will become the primary operational mode for AuxTel. 
+The Survey Mode utilizes semi-autonomous, scheduler-driven operations and the roles, responsibilities, and expectations for this mode are defined. 
 
-Add content here
-================
+Guidelines for Operations
+=========================
 
-Add content here.
-See the `reStructuredText Style Guide <https://developer.lsst.io/restructuredtext/style.html>`__ to learn how to create sections, links, images, tables, equations, and more.
+Survey mode is intended to be the primary mode for Auxiliary Telescope (AuxTel) operations. 
+It is a passive mode of operations, with nighttime observations driven by the Scheduler with minimal staff intervention. 
+The central tenant of operations in this mode is that the AuxTel operations are secondary to any responsibilities the summit staff may have related to Simonyi Survey Telescope operations. 
+
+This results in the following guidelines for operations in survey mode: 
+
+- The observing staff are not be required to actively monitor or report on incoming data quality.
+- Logging will be minimal; limited to short narrative statements needed to report on changes during the night or provide context for faults and recovery attempts.
+- Faults will be recovered when observing staff have available time, and the AuxTel may remain in Standby state during the night for extended periods of time. 
```

Comment:
We define fault loss as time between the beginning of the symptom that stopped us from taking images until we take the next image. Letting things sit will over-inflate this statistic. I think we should be mindful of how we decide to qualify it, because we have been collecting down time reports on AT for the last year+. If it's not a metric we care about, it will be heard to re-learn that later.

Reply:
I see what you mean about having the fault loss look inflated if faults are allowed to sit w/o being addressed - but I think that's a feature not a bug of this ops model.  For time accounting it may be useful to differentiate between "time lost while nobody was paying attention to AuxTel" vs. "time lost while actively attempting to recover AuxTel".  For the purposes of this document review, I think line 30 is fine.

Reply:
Both good points, I'm kind of in-between. I think we can keep tracking the original definition of Fault Loss (time between exposures), but it mean something totally different now. If it turns out we have 6hrs per night of Fault Loss because we cannot get to the AuxTel in time, we will probably have to consider something like shutting down AuxTel or updating our operational model. 

I'd rather not ask people to try and differentiate between "time lost while nobody was paying attention to AuxTel" vs. "time lost while actively attempting to recover AuxTel" since that just adds to the pile of things to keep track of. 

I added a disclaimer about the inflated time loss here, let me know if that covers both points.

Reply:
Word, Erik D. I think that's a good response. I like thinking about this as a mechanism to drive AuxTel's future - if it's going to need resources devoted to it or not.